doc/API.md

```diff
@@ -217,3 +216,36 @@ This command does not return anything for now.
 
 | Field          | Type      | Description                                          |
 | -------------- | --------- | ---------------------------------------------------- |
+
+### `gethistory`
+
+`gethistory` retrieves a paginated list of accounting events.
+
+Aiming at giving an accounting point of view, the amounts returned by this call are the total
+of inflows and outflows net of any change amount (that is technically a transaction output, but not a cash outflow).
+
+#### Request
+
+| Field         | Type         | Description                                                          |
+| ------------- | ------------ | -------------------------------------------------------------------- |
+| `start`       | int          | Timestamp of the beginning of the period to retrieve events for      |
+| `end`         | int          | Timestamp of the end of the period to retrieve events for            |
+| `limit`       | int          | Maximum number of events to retrieve                                 |
+
+#### Response
+
+| Field          | Type   | Description                                |
+| -------------- | ------ | ------------------------------------------ |
+| `events`       | array  | Array of [Event resource](#event-resource) |
+
+##### Event Resource
+
+| Field         | Type          | Description                                                                                                             |
+| ------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `blockheight` | int           | Blockheight of the event transaction                                                                                    |
+| `kind`        | string        | Type of the event. Can be `receive`, `spend`                                                                            |
+| `date`        | int           | Timestamp of the event                                                                                                  |
+| `amount`      | int           | Absolute amount in satoshis that is entering or exiting the wallet                                                      |
+| `miner_fee`   | int or `null` | Total of the miner fees caused by the operation, `null` if the event is a `receive` event                               |
+| `tx`          | string        | bitcoin transaction of the spend, `null` if the event is a `receive` event                                              |
+| `outpoint`    | string        | Hex string of the output created by this `receive` event, `null` if the event is a `spend` event                        |
```

Comment:
To be honest this feels very clumsy... It looks like everything that was needed for the home screen on the GUI was stuffed in this call, whatever it could possibly look like in the end. I don't like this way of designing an API, but i appreciate the schedule is tight.